lib/puppet_litmus/serverspec.rb

```diff
@@ -1,14 +1,29 @@
 # frozen_string_literal: true
 
-# helper functions for running puppet commands, and helpers
+# helper functions for running puppet commands. They execute a target system specified by ENV['TARGET_HOST']
 module PuppetLitmus::Serverspec
+  # Applies a manifest twice. First checking for errors. Secondly to make sure no changes occur.
+  #
+  # @param manifest [String] puppet manifest code to be applied.
+  # @return [Boolean] The result of the 2 apply manifests.
   def idempotent_apply(manifest)
     manifest_file_location = create_manifest_file(manifest)
     apply_manifest(nil, catch_failures: true, manifest_file_location: manifest_file_location)
     apply_manifest(nil, catch_changes: true, manifest_file_location: manifest_file_location)
   end
 
+  # rubocop:disable Layout/TrailingWhitespace
```

Comment:
Is the trailing whitespace needed? I think that in Puppet Strings (which is also Yard based) it isn't needed. Or is it to force a newline?

Reply:
it is to force the newline, to make it easier to read.